url/interfaces.any.js

```diff
@@ -28,7 +21,7 @@
   USVString toJSON();
 };
 
-[Constructor(optional (USVString or URLSearchParams) init = ""),
+[Constructor(optional (sequence<sequence<USVString>> or USVString) init = ""), // or record<USVString, USVString>
```

Comment:
The webidl2.js in the tree doesn't seem to bear any similarity to the one in w3c/webidl2.js.  It looks like it's based on the master branch instead of the develop branch.  I have no idea how all this submodule magic is supposed to work, so I don't know how to fix it.  @jgraham?